packages/fast-components-react-base/src/dialog/dialog.tsx

```diff
@@ -50,6 +53,15 @@ class Dialog extends Foundation<IDialogHandledProps & IManagedClasses<IDialogCla
         );
     }
 
+    /**
+     * React life-cycle method
+     */
+    public componentDidMount(): void {
+        if (canUseDOM() && this.props.onDismiss) {
+            window.addEventListener("keydown", this.onDialogKeyboardDismiss);
```

Comment:
Don't we need to remove the event listener on unMount?

Reply:
yep - updated.